lib/src/presentation/scenes/habit/habit_row.dart

```diff
@@ -77,6 +79,23 @@ class _HabitRowState extends State<HabitRow> {
           ],
         ),
       );
+    } else {
+      return Container(
+        padding: EdgeInsets.symmetric(horizontal: _screenSize.width * 0.015),
+        height: _screenSize.height * 0.07,
+        color: XHColors.darkGrey,
+        child: Padding(
+          padding: const EdgeInsets.only(left: 40.0),
+          child: Row(
+            children: <Widget>[
+              _habitTitle(habitState.habitTitle),
+              _marks(habitState.checkedDays, _weekDays),
+            ],
```

Comment:
@phenybutum can we place check if habit has end date here to avoid code duplication?